kitchen/ph5tomsAPI.py

```diff
@@ -228,8 +228,8 @@ def filenamesacimg_gen(self, stream):
             ret = os.path.join(self.out_dir, "preview_images", ret)
         return ret
     
-    @staticmethod
-    def get_nonrestricted_segments(station_to_cut_list, restricted, station_to_cut_segments=[]):
+    @classmethod
+    def get_nonrestricted_segments(cls, station_to_cut_list, restricted, station_to_cut_segments=[]):

```

Comment:
This change is unrelated to #19. I just changed this method to use the classmethod annotation to be consistent across APIs.